queries.yaml

```diff
@@ -454,6 +454,68 @@ post_process:
     params:
       layer: roads
 
+  - fn: vectordatasource.transform.point_in_country_logic
+    params:
+      layer: pois
+      country_layer: admin_areas
+      country_code_attr: iso_code
+      output_attr: drives_on_left
+      where: kind == 'mini_roundabout'
+      # see https://en.wikipedia.org/wiki/List_of_countries_with_left-hand_traffic
+      logic_table:
+        AU: true
```

Comment:
File under `the sun never set on the British Empire`...

I checked a few more sources and came up with 24 more to add:

iso_a2 | country
------ | -------
AI | Anguilla
AG | Antigua and Barbuda
LC | Saint Lucia
JM | Jamaica
VG | British Virgin Islands
KN | Saint Kitts and Nevis
VI | U.S. Virgin Islands
KY | Cayman Islands
VC | Saint Vincent and the Grenadines
TC | Turks and Caicos Islands
TT | Trinidad and Tobago
SR | Suriname
GS | South Georgia and the South Sandwich Islands
GY | Guyana
SH | Saint Helena, Ascension and Tristan da Cunha
GD | Grenada
FK | Falkland Islands
SC | Seychelles
DM | Dominica
BS | Bahamas
BM | Bermuda
MU | Mauritius
BB | Barbados
MS | Montserrat

Here's my full list iso_a2 code and country name:


iso_a2 | country | note | new
------ | ------- | ---- | ---
AG | Antigua and Barbuda |  | 1
AI | Anguilla |  | 1
AU | Australia |  | 
BB | Barbados |  | 1
BD | Bangladesh |  | 
BM | Bermuda |  | 1
BN | Brunei |  | 
BS | Bahamas |  | 1
BT | Bhutan |  | 
BW | Botswana |  | 
CC | Cocos (Keeling) Islands |  | 
CK | Cook Islands |  | 
CX | Christmas Island |  | 
CY | Cyprus |  | 
CY | Northern Cyprus | fake ISO code | 
DM | Dominica |  | 1
FJ | Fiji |  | 
FK | Falkland Islands |  | 1
GB | United Kingdom |  | 
GD | Grenada |  | 1
GG | Guernsey |  | 
GS | South Georgia and the South Sandwich Islands |  | 1
GY | Guyana |  | 1
HK | Hong Kong |  | 
ID | Indonesia |  | 
IE | Ireland |  | 
IM | Isle of Man |  | 
IN | India |  | 
JE | Jersey |  | 
JM | Jamaica |  | 1
JP | Japan |  | 
KE | Kenya |  | 
KI | Kiribati |  | 
KN | Saint Kitts and Nevis |  | 1
KY | Cayman Islands |  | 1
LC | Saint Lucia |  | 1
LK | Sri Lanka |  | 
LS | Lesotho |  | 
MO | Macau | fake ISO code | 
MS | Montserrat |  | 1
MT | Malta |  | 
MU | Mauritius |  | 1
MV | Maldives |  | 
MW | Malawi |  | 
MY | Malaysia |  | 
MZ | Mozambique |  | 
NA | Namibia |  | 
NF | Norfolk Island |  | 
NP | Nepal |  | 
NR | Nauru |  | 
NU | Niue |  | 
NZ | New Zealand |  | 
PG | Papua New Guinea |  | 
PK | Pakistan |  | 
PN | Pitcairn Islands |  | 
SB | Solomon Islands |  | 
SC | Seychelles |  | 1
SG | Singapore |  | 
SH | "Saint Helena |  Ascension and Tristan da Cunha" |  | 1
SR | Suriname |  | 1
SZ | Swaziland |  | 
TC | Turks and Caicos Islands |  | 1
TH | Thailand |  | 
TK | Tokelau |  | 
TL | East Timor |  | 
TO | Tonga |  | 
TT | Trinidad and Tobago |  | 1
TV | Tuvalu |  | 
TZ | Tanzania |  | 
UG | Uganda |  | 
VC | Saint Vincent and the Grenadines |  | 1
VG | British Virgin Islands |  | 1
VI | U.S. Virgin Islands |  | 1
WS | Samoa |  | 
ZA | South Africa |  | 
ZM | Zambia |  | 
ZW | Zimbabwe |  |